apps/portal/src/content/docs/components/avatar.mdx

```diff
@@ -4,150 +4,171 @@ description: Avatar component
 beta: true
 ---
 
-The Avatar component provides a way to represent an image with a fallback for when the image can not be loaded.
+The Avatar component provides a way to represent an image with a fallback for when the image cannot be loaded. It also supports displaying initials derived from the `name` prop.
 
 import { DocsPage } from "../../../components/docs-page";
 
+### Image (Default)
```

Comment:
I think we can get rid of most of these sub-sections and just have a single example which shows all combinations in a grid to better align with the other component docs.